### ISSUE TITLE
sidebery issue 

### ISSUE THREAD

**Issue by pizzav-xyz** [artsyfriedchicken/EdgyArc-fr#48]
the sideberry is not changing when i apply your config
![image](https://github.com/artsyfriedchicken/EdgyArc-fr/assets/103120356/e968ff23-7387-4859-91b8-9b517a997e0f)

**Comment by artsyfriedchicken** [comment]
please make sure you have `af.sidebery.edgyarc-theme` enabled!!

**Comment by pizzav-xyz** [comment]
> please make sure you have `af.sidebery.edgyarc-theme` enabled!!

thanks, any other things i should enable?

**Comment by artsyfriedchicken** [comment]
> please make sure you have `af.sidebery.edgyarc-theme` enabled!!

Does this not solve your issue? it should!

**Comment by kittenvr** [comment]
Yeah but I want to know if there are other things that make the theme
better like that

**Comment by pizzav-xyz** [comment]
Yeah but I want to know if there are other things that make the theme better like that

_Sorry if I sent it multiple times_
